packages/kernel-6.12/bootconfig-aws.conf

```diff
@@ -0,0 +1,2 @@
+kernel.initcall_blacklist = vmd_drv_init, megasas_init, mpt3sas_init, pqi_init
```

Comment:
This line shouldn't be necessary for either aws or vmware snippets if we're not building the additional metal-only drivers.